src/SleepRepository.js

```diff
@@ -55,6 +59,11 @@ class SleepRepository {
     let userWithMostHoursSlept = sortedByHoursSlept[sortedByHoursSlept.length - 1]
     return sortedByHoursSlept.filter(instance => instance.hoursSlept === userWithMostHoursSlept.hoursSlept);
   }
+
+  createCurrentUser(id) {
```

Comment:
This method is incredibly useful since it allows us to generate a single sleep object based on the ID. This user can act as our logged in user so we can display their data on the page. Good stuff.